api-2.0/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirConditionDaoImpl_2_0.java

```diff
@@ -26,7 +26,7 @@
 @Primary
 @Component
 @Setter(AccessLevel.PACKAGE)
-@OpenmrsProfile(openmrsPlatformVersion = "2.0.* - 2.1.*")
+@OpenmrsProfile(openmrsPlatformVersion = "2.0.* - 2.0.*")
```

Comment:
According to your PR description that means that this versions will expand to 2.1.0 on the upperBound, does this mean that the other versions 2.1.* will not be supported apart from 2.1.0?
This question applies to the other parts of the code with this change...

Reply:
Hmmm... yes, you make a good point. I'm going to revert this to just apply the necessary future dating so things with run in 2.4.0+. Thanks!